### PR TITLE
refs #50 

### PR DESCRIPTION
**Pull request by emullaney**
added region field to case study and marketplace entry models & templates, made snippet-able

### CODE REVIEW

**Comment by emullaney** [comment]
Note: remove lat/long from Region

File: portal_pages/models.py [hunk 1]
```diff
@@ -259,7 +273,8 @@ def marketplace_index(self):
     MultiFieldPanel(ContactFields.panels, "Contact"),
     InlinePanel(MarketplaceEntryPage, 'services', label="Services"),
     InlinePanel(MarketplaceEntryPage, 'expertise_tags', label="Expertise"),
-    InlinePanel(MarketplaceEntryPage, 'countries', label="Locations of Expertise"),
+    InlinePanel(MarketplaceEntryPage, 'regions', label="Regions of Expertise"),
```
Comment (kmtracey) [inline portal_pages/models.py]: "Regions of Experience" is what I recall from the meeting and what we have in notes.

Comment (emullaney) [inline portal_pages/models.py]: For that matter, it's also Countries of Experience.

**Comment by kmtracey** [comment]
:+1:

**Pull request merged**